tests/sanity/test/apiml/test-api-zosmf-via-gateway.js

```diff
@@ -31,13 +31,13 @@ describe('test api mediation layer zosmf api', function() {
     token = await apiml.login();
   });
 
-  it('should be able to get z/OS Info via the gateway port and endpoint (/api/v1/zosmf/info)', async function() {
+  it('should be able to get z/OS Info via the gateway port and endpoint (/ibmzosmf/api/v1/info)', async function() {
```

Comment:
To ibmzosmf it changed years ago, the zosmf was kept only for backwards compatibility. 

But you are correct, it needs to be added to the breaking changes that we intend to remove the zosmf

Reply:
@balhar-jakub - isn't `zosmf/info` part of the endpoint path? https://docs.zowe.org/v2.6.x/extend/extend-apiml/api-mediation-components-of-url/#url-with-empty-basepath

Although it perhaps seems redundant, `zosmf` is part of the API path and applications use it that way ([such as Zowe CLI](https://github.com/zowe/zowe-cli/blob/master/packages/zosjobs/src/JobsConstants.ts#L112)).

Is this change really necessary?

Reply:
Would this mean that the swagger doc in z/OSMF UI (which includes `zosmf`) would look different than the swagger docc for z/OSMF when viewed through the API ML catalog UI? 

IBM z/OSMF:
![image](https://github.com/zowe/zowe-install-packaging/assets/14940561/d4635615-208a-4cd9-acfc-2fcb62e370cc)

Reply:
It looks like the test has been defined this way since March 2022: [commit](https://github.com/zowe/zowe-install-packaging/commit/0ca96cddcfe9326456f26ca075127ca2de644705#diff-d851d7cd9626c7d890cd7305ab614dcfdb99f2747e4667eb3ac6ba37b34d9a93)